### ISSUE TITLE
Posts (composed on iOS?) sometimes have P SEP and L SEP unicode characters in them.

### ISSUE THREAD

**Issue by AlanBell** [mastodon/mastodon#21851]
### Steps to reproduce the problem

By way of example
<img width="157" alt="image" src="https://user-images.githubusercontent.com/691857/204566289-d00f0394-b5e0-40c7-a3e3-cb6640fb695e.png">
these broken characters are the unicode paragraph and line separators. U+2028 and U+2029
I am not entirely clear how these come about, but I think it is a recent iOS and viewed in Chrome on Windows. The intention appears to be that these are newline \n character equivalents.

### Expected behaviour

show post with newlines

### Actual behaviour

shows broken unicode characters

### Specifications

Mastodon 4

**Comment by AlanBell** [comment]
might be related to copying and pasting from other applications

**Comment by rbairwell** [comment]
Ah, those [Mojibake](https://en.wikipedia.org/wiki/Mojibake) characters are (as you rightly pointed out) are a Paragraph Separator ([PSEP](https://www.fileformat.info/info/unicode/char/2029/index.htm)) and Line Separator ([LSEP](https://www.fileformat.info/info/unicode/char/2028/index.htm)) - as the "solution" to the conflicting line breaks across multiple platforms was, of course, invent a new standard (obl: https://xkcd.com/927/ )

By looking at reports - this is either a [Chrome](https://stackoverflow.com/questions/39603446/why-is-this-lsep-symbol-showing-up-on-chrome-and-not-firefox-or-edge) issue or a [Windows](https://answers.microsoft.com/en-us/msoffice/forum/all/in-ms-word-getting-l-sep-or-lsep-boxes-showing-up/489554fe-98cd-49b3-abfe-9d6c3ba18500) issue - but it'll come down to which default browser font you have (Times/Times New Roman, Helvetica, Arial, Courier and Courier New on Windows does NOT have those glyphs).

Basically - I don't think there's anything really Mastodon should (or could) do to this. If you switch browser fonts to one listed on say https://www.fileformat.info/info/unicode/char/2029/fontsupport.htm , then it should render the characters okay across all websites.

**Comment by AlanBell** [comment]
Changing the font does change whether the mojibake characters display, but it doesn't appear to render line breaks. I suspect that line breaks in the HTML get ignored like other line breaks in HTML - so for example source HTML like this:
```
<p>this paragraph
is all on one line</p>

```

gets rendered as one line regardless of whether the line break is \n or \r\n or  U+2029
I think what we should be doing is rendering U+2029 and/or U+2028 as <br> (and arguably <p></p> but there isn't an end separator for paragraphs I think)
